widget/assets/css/top3.css

```diff
@@ -0,0 +1,24 @@
+.layer-border-left {
+    border-radius: 4px;
+    height: 80px;
+    width: 350px;
+    margin-top: 20px;
```

Comment:
please use pre-defined class for margin, padding, display

for margin-top: 20px use margin-top-20 class
for display: block use display-block class
for margin-left use margin-left-md class

this way we can save number of lines of repetitive code